src/app/pages/storage/storage.page.scss

```diff
@@ -1,7 +1,43 @@
-ion-thumbnail {
-  --size: 25vw;
+
+.img-card {
+  margin: calc( var( --ion-padding, 9px ) / 3 );
+  width: 30vw;
+
+}
+
+.Title {
+  color: #000;
 }
 
-ion-card {
-  margin: calc(var(--ion-padding, 16px) / 4);
+.setTitle {
+  display: flex;
+  flex-direction: row;
+  justify-self: start;
+  padding-bottom: 10px;
+  color: #000;
+  margin: calc( var( --ion-padding, 9px ) / 3 );
+}
+
+.img-thumbnail {
+  --size: calc( 90vw / 3 );
```

Comment:
Codacy found an issue: [Unexpected whitespace after "(" in a single-line function (function-parentheses-space-inside)](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6209007)